file-formats/dobj/type/zif_aff_dobj_v1.intf.abap

```diff
@@ -0,0 +1,57 @@
+INTERFACE zif_aff_dobj_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Structure Definition Details</p>
+    "! Structure definition details
+    BEGIN OF ty_structure_definition,
+      "! <p class="shorttext">Item</p>
+      "! Item
+      "! $required
+      item      TYPE n LENGTH 4,
+      "! <p class="shorttext">Parent Table</p>
+      "! Parent table
+      "! $required
+      parent_table    TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Dependent Table</p>
+      "! Dependent table
+      "! $required
+      dependent_table TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_structure_definition,
+    "! <p class="shorttext">Structure Definitions</p>
+    "! Structure definitions
+    ty_structure_definitions TYPE SORTED TABLE OF  ty_structure_definition WITH UNIQUE KEY item.
+
+
+  TYPES:
+    "! <p class="shorttext">Data Destruction Object Details</p>
+    "! $required
+    BEGIN OF ty_dobj_details,
```

Comment:
still struggling with the name.
what about just "details" or "object_details"

Reply:
was wrong. everything is fine, just the example is not up to date

Reply:
> was wrong. everything is fine, just the example is not up to date 

I have mad a separate suggestion to update the example file

However, I would also update the title to "Attribute Details"